src/client/pages/OfferNew/Checkout/CreditCheckInfo.tsx

```diff
@@ -0,0 +1,50 @@
+import styled from '@emotion/styled'
+import { colorsV2, colorsV3 } from '@hedviginsurance/brand'
+import React from 'react'
+import { FadeIn } from 'components/animations/appearings'
+import { InfoIcon } from 'components/icons/Info'
+import { useTextKeys } from 'utils/textKeys'
+
+const InfoBox = styled.div`
+  padding: 1rem;
+  padding-left: 3rem;
+  margin-bottom: 1.5rem;
+  background-color: ${colorsV2.violet100};
```

Comment:
We only use `colorsV3` :) 
```suggestion
  background-color: ${colorsV3.purple100};
```

Reply:
But it's not the same 😰  🙈   In the design, it was `#efe8ff`. But I see your point, I'll let Zak know